publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java

```diff
@@ -227,6 +228,21 @@ void shouldPersistProvidedStatusWhenMachineUserPersistsPublicationWithAnyStatus(
         assertThat(publicationResponse.getStatus(), is(equalTo(status)));
     }
 
+    @Test
+    void shouldSetPublishedDateWhenMachineUserPersistsPublicationWithStatusPublished()
+        throws Exception {
```

Comment:
There are test cases that verify that not machine user is not able to create publication with other statuses than DRAFT.